packages/core/src/sync/sync-api.ts

```diff
@@ -51,22 +52,37 @@ export type SyncConfig = {
 
 // TODO (CDB-2106): move to SyncStatus Class
 export interface ActiveSyncStatus {
+  // The block the sync starts at
+  startBlock: number
+  // The block the sync is currently processing
   currentBlock: number
+  // The block the sync will end on
   endBlock: number
+  // Models that are being synced
   models: Array<string>
+  // Date when the sync was requested
   created: Date
```

Comment:
nitpick but the naming is inconsistent between `created` and `startTime`.  Maybe rename `created` to `createdTime` for symmetry?

Reply:
or `createdAt` and `startedAt`?